README.md

```diff
@@ -1,99 +1,59 @@
-# Role Name
+# keylime_server
+------------------
+
+Ansible role for configuring and deploying the server components for Keylime Remote Attestation.
+
+# Supported Distributions
```

Comment:
```suggestion
## Supported Distributions
```
Can have only 1 top level `#` heading in markdown

Reply:
Thanks, fixed.